marlowe-marketplace-client/plutus-pab.yaml

```diff
@@ -0,0 +1,45 @@
+dbConfig:
```

Comment:
Not sure what this file does, I think it's related for deployed so I left it for later.
cc @shmish111